docs/tutorials/continuing_runs.md

```diff
@@ -61,6 +61,20 @@ We use `--restart 5` because the first new step is the sixth in the new
 workflow: the five initial steps and three new steps. Remember `--restart` is
 0-indexed.
 
+
+## Extend a run
+
+You can extend a successful run with additional steps. For that, prepare a
+configurtation file containing only the new steps you wish to execute on top of
+a previously successful run. In these cases, you **don't** need to define the
+`run_dir` and `molecules` parameters in this new configuration file. You **can**
```

Comment:
Just a curiosity: Does this mean that it is forbidden to define run_dir and molecules or just that they are not mandatory?

Reply:
In case they are forbidden I would substitute `don't need to` with `must not`

Reply:
They will be ignored. Should I add that?